### PR TITLE
Document correction

### PR DESCRIPTION
**Pull request by sumiyoshi**
Is this incorrect?

### CODE REVIEW

**Comment by essen** [comment]
Yep it hasn't been updated yet.

**Comment by sumiyoshi** [comment]
Thank you for reply

File: doc/src/guide/handlers.asciidoc [hunk 1]
```diff
@@ -29,9 +29,9 @@ We need to use the Req object to reply.
 [source,erlang]
 ----
 init(Req0, State) ->
-    Req = cowboy_req:reply(200, [
+    Req = cowboy_req:reply(200, #{
         {<<"content-type">>, <<"text/plain">>}
```
Comment (essen) [inline doc/src/guide/handlers.asciidoc]: Yep!

Comment (sumiyoshi) [inline doc/src/guide/handlers.asciidoc]: Is not ->, =>?

**Comment by essen** [comment]
Still not quite right, but don't worry I will take care of it. Almost finished updating the cowboy_req manual.

**Comment by essen** [comment]
Do leave open so I don't forget. :-) Cheers.

**Comment by sumiyoshi** [comment]
OK！

**Comment by essen** [comment]
This is done on my local copy and will be pushed later today. Thanks!

**Pull request closed without merging**